rootfs/api/models/app.py

```diff
@@ -33,6 +34,26 @@
 logger = logging.getLogger(__name__)
 
 
+def is_experimental_native_ingress():
+    """
+    Will determine the value of the experimental_native_ingress flag
+    """
+    is_ingress = os.getenv('EXPERIMENTAL_NATIVE_INGRESS', False)
+    if not is_ingress:
+        return False
+    return True
+
+
+def get_experimental_native_ingress_hostname():
+    """
+    Will return the experimental native ingress platform_domain value, or an empty string
```

Comment:
This docstring doesn't match the code. Instead maybe:
"Returns the experimental native ingress hostname or raises ServiceUnavailable if it's not set."